reactor-core/src/jcstress/java/reactor/core/publisher/SinkManySerializedStressTest.java

```diff
@@ -45,7 +45,7 @@
 	@Outcome(id = {"FAIL_NON_SERIALIZED, OK, 1"}, expect = ACCEPTABLE, desc = "second wins")
 	@Outcome(id = {"OK, OK, 2"}, expect = ACCEPTABLE, desc = "one after another")
 	@State
-	public static class TryEmitNextStressTest extends SerializedManySinkStressTest {
+	public static class TryEmitNextManySerializedStressTest extends SinkManySerializedStressTest {
```

Comment:
I am wondering why IDEA renamed this class :D I think we should rename it back to keep the inner class names shorter

Reply:
to clarify, this should indeed be renamed back to `TryEmitNextStressTest`